### ISSUE TITLE
What is exactly this thing ?

### ISSUE THREAD

**Issue by dsslimshaddy** [NARKOZ/hacker-scripts#145]
I dont understand what is this thing about ? what it does ?whats its use ?

**Comment by SirWindfield** [comment]
These are a bunch of code snippets used by some russian coder (if I remember right) for automating various tasks during his work. He was lazy and tried to automate a lot of stuff, like brewing his coffee etc.

**Comment by dsslimshaddy** [comment]
Why its so starred its easy to do . U just need a microcontroller circuit and some stuffs mainly focused on electrical stuffs. but programing part is easy as kiddie

**Comment by SirWindfield** [comment]
it got viral on reddit and other platforms at the same time. That's probably one of the reasons

**Comment by Tarrasque18** [comment]
Is it possible for me to make my own scripts for my own stuff?

**Comment by phreakyphoenix** [comment]
Yeah it sure is, and if you think any of your scripts will make lives easier for energy-saving devs, feel free to make a pull request.

**Comment by kaneda** [comment]
You should read the blog post from the README: https://www.jitbit.com/alexblog/249-now-thats-what-i-call-a-hacker/

**Comment by bluaay** [comment]
! Ox
